CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 xxxxxxxxxxxxxxxxxxx
 ===================
+- Added `Apify.amIAtHome()` function that returns `true` when code is running at Apify platform and `false` otherwise
```

Comment:
`amIAtHome` looks somehow weird, how about calling it `isAtHome` ? That's consistent to APIFY_IS_AT_HOME env var.